docs/pkgdown.yml

```diff
@@ -1,4 +1,4 @@
-pandoc: '2.6'
+pandoc: 2.3.1
```

Comment:
Just curious ... Why is this specific version of `pandoc` needed?

Reply:
It isn't - the knitr run copies that into the file for documentation.